sortinghat/core/api.py

```diff
@@ -805,34 +805,34 @@ def withdraw(ctx, uuid, organization, from_date=None, to_date=None):
 
     try:
         if min_range < from_date:
-            add_enrollment(trxl, uidentity, org, start=min_range, end=from_date)
+            add_enrollment(trxl, individual, org, start=min_range, end=from_date)
         if max_range > to_date:
-            add_enrollment(trxl, uidentity, org, start=to_date, end=max_range)
+            add_enrollment(trxl, individual, org, start=to_date, end=max_range)
     except ValueError as e:
         raise InvalidValueError(msg=str(e))
 
-    uidentity.refresh_from_db()
+    individual.refresh_from_db()
 
     trxl.close()
 
-    return uidentity
+    return individual
 
 
 @django.db.transaction.atomic
 def merge_identities(ctx, from_uuids, to_uuid):
```

Comment:
As I said before, I'd like to keep the concept of identities instead of individuals because the access with be through identities. Individuals might be just something internal to organize the information.

Reply:
I don't fully agree in this case, because you are not merging identities as you are not modifying any of their information, but the information about the individuals pointing them (enrollments, profile). WDYT?

Reply:
After last @sduenas comments we agreed on leave this for the moment, as there are more changes to come.